README.md

```diff
@@ -146,29 +176,33 @@ public class GithubRepoLoader extends DefaultLoader<GithubRepoLoader.RepoDefinit
      * See "Multiple params in API Call"
      * section in GithubRepoLoader
      */
-    public class RepoDefinition
+    public class RepoDefinitionParams implements LoaderParams
```

Comment:
can we change `RepoDefinitionParams` to just `RepoParams`?

I made an earlier decision to refer to these as `Definitions`, but now we switched over to Just simply `Params`.